src/app/aircraft/edit/index.njk

```diff
@@ -3,11 +3,11 @@
 {% import "common/templates/includes/macros.njk" as m %}
 
 {% block pageTitle %}
-	GAR | Edit an Aircraft
+	{{ __('error_prefix') if errors }} GAR | Edit an Aircraft
```

Comment:
When an error appears, screen reader can read the title of the page and it will say "Error: " to inform the user straight away.